core/src/main/java/hudson/PluginManager.java

```diff
@@ -1557,6 +1557,10 @@ public HttpResponse doPluginsSearch(@QueryParameter String query, @QueryParamete
                         releaseTimestamp.put("displayValue", Messages.PluginManager_ago(Functions.getTimeSpanString(plugin.releaseTimestamp)));
                         jsonObject.put("releaseTimestamp", releaseTimestamp);
                     }
+                    if (plugin.healthScore != null) {
+                        jsonObject.put("healthScore", plugin.healthScore);
+                        jsonObject.put("healthScoreClazz", plugin.healthScoreClazz);
```

Comment:
```suggestion
                        jsonObject.put("healthScoreClass", plugin.healthScoreClass);
```

Plus everywhere else.

There's no need for the `clazz` workaround unless you actually just call it `clazz`, as `class` is a Java keyword.

Reply:
Force of habit. I'll fix that.